### PR TITLE
convert development package to runtime package

### PR DESCRIPTION
**Pull request by jiaqizhao122**

### CODE REVIEW

File: programs/kbuild/pkg/depends
Comment (rli9) [inline programs/kbuild/pkg/depends:1]: skip the change of this test, since kbuild does make kernel.

File: programs/build-ceph/pkg/depends
Comment (rli9) [inline programs/build-ceph/pkg/depends:1]: skip this, which does build ceph

File: programs/kbuild/pkg/depends
Comment (jiaqizhao122) [reply programs/kbuild/pkg/depends:1]: ok

File: programs/build-ceph/pkg/depends
Comment (jiaqizhao122) [reply programs/build-ceph/pkg/depends:1]: ok

**Comment by rli9** [comment]
thanks for the patch

**Pull request merged**